apis/fluent/atrium-api-fluent/src/commonMain/kotlin/ch/tutteli/atrium/api/fluent/en_GB/listExpectations.kt

```diff
@@ -0,0 +1,20 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.logic._logic
+import ch.tutteli.atrium.logic.last
+
+/**
+ * Expects that the subject of `this` expectation (a [List]) is not empty and that the last element
+ * in the list holds all assertions the given [IndexWithCreator.assertionCreator] creates for it.
+ *
+ *  Use the function `last() { ... }` to create an [IndexWithCreator].
+ *
+ * @return an [Expect] for the subject of `this` expectation.
+ *
+ * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.ListFeatureExtractorSamples.last
+ *
+ * @since 1.2.0
+ */
+fun <E, T : List<E>> Expect<T>.last(assertionCreator: Expect<E>.() -> Unit): Expect<T> =
```

Comment:
You forgot to add the val (first point in the issue) => found it in the correct file.
Sorry, copy&paste error on my side in the issue descriptoin, also this function should be in listFeatureExtractors.kt

Reply:
Not sure if I got you right. As a result I removed `listExpectations.kt` totally, can you please have a look on changes?

Reply:
@constantine-zubrilin Almost there, a few little things and we are good